pkgs/tools/system/plan9port/default.nix

```
@@ -37,8 +48,41 @@ stdenv.mkDerivation rec {
     DarwinTools
   ]);
 
-  builder = ./builder.sh;
-  libXt_dev = libXt.dev;
+  configurePhase = ''
```

When using custom `configurePhase`, decorate it with hooks:

```suggestion
  configurePhase = ''
    runHook preConfigure
```